client/ayon_nuke/plugins/load/load_clip.py

```diff
@@ -422,6 +422,7 @@ def set_colorspace_to_node(
         ):
             self.log.info(f"Used colorspace: {used_colorspace}")
             read_node["colorspace"].setValue(used_colorspace)
+            read_node["raw"].setValue(False)
```

Comment:
Since RAW is by default set to `False` it seems that your `init.py` somewhere might be overriding it. Please check your configuration.

Reply:
you were absolutely right! Not sure how I missed it. Will close the PR